docs/development/design-decisions.md

```diff
@@ -5,64 +5,81 @@ This file documents the design choices as well as configuration options.
 
 ## Separate Branches per dependency
 
-By default, `renovate` will maintain separate branches per-dependency.
+By default, `renovate` will maintain separate branches for each dependency.
 So if 20 dependencies need updating, there will be at least 20 branches/PRs.
-Although this may seem undesirable, it was considered even less desirable if all 20 were in the same Pull Request and it's up to the users to determine which dependency upgrade(s) caused the build to fail.
+Although this may seem undesirable, it's even less desirable if all 20 were in the same Pull Request.
+That way it would be up to the users to determine which dependency upgrade(s) caused the build to fail.
 
-However, it's still possible to override the default branch and PR name templates in such a way to produce a single branch for all dependencies.
+However, you can override the default branch and PR name templates to get a single branch for all dependencies.
 The `groupName` configuration option can be used at a repository level (e.g. give it the value `All`) and then all dependency updates will be in the same branch/PR.
 
 ## Separate Minor and Major PRs
 
 `renovate` will create multiple branches/PRs if both major and minor branch upgrades are available.
 For example, if the current example is 1.6.0 and upgrades to 1.7.0 and 2.0.0 exist, then `renovate` will raise PRs for both the 1.x upgrade(s) and 2.x upgrade(s).
 
+Our reasons for separating minor and major PRs:
+
 - It's often the case that projects can't upgrade major dependency versions immediately
 - It's also often the case that previous major versions continue receiving Minor or Patch updates
 - Projects should get Minor and Patch updates for their current Major release even if a new Major release exists
 
-This can be overridden via the config option `separateMajorMinor`.
+This behaviour can be overridden via the config option `separateMajorMinor`.
 
 ## Branch naming
 
-Branches are named like `renovate/webpack-1.x` instead of `renovate/webpack-1.2.0`.
+Branches have names like `renovate/webpack-1.x` instead of `renovate/webpack-1.2.0`.
+
+We do this because:
 
 - Branches often receive updates (e.g. new patches) before they're merged
 - Naming the branch like `1.x` means its name still names sense if a `1.2.1` release happens
 
-Note: Branch names are configurable using string templates.
+Note: You can configure the branch names by using a string template.
 
 ## Pull Request Recreation
 
-By default, the script does not create a new PR if it finds an identical one already closed.
+By default, the script does not create a new PR if it finds an identical PR that's already closed.
 This allows users to close unwelcome upgrade PRs and not worry about them being recreated every run.
-Typically this is most useful for major upgrades.
-This option is configurable.
+Typically, this is most useful for major upgrades.
+This option is configurable. (WHERE/HOW???)
```

Comment:
If we're only saying, this option is configurable, it's not very much help without saying where/how this can be configured, I think?